### DISCUSSION TITLE
WebRTC port range

### DISCUSSION THREAD

**Discussion by cemaka** [OvenMediaLabs/OvenMediaEngine#1396] [Help]
Hi all,

TL;DR: I need to learn and set port range that OME uses for webrtc traffic.

----

I need to stream video over webrtc through OME to OME Player on the browser. However, due to network topology, OME server and viewer can not establish connection directly. OME server (S) and viewer (V) are on the separate networks. There is another computer in the middle (M),  which has connections to both networks. Viewer establishes connection to OME server through TCP proxy (HAProxy) on M. HTTP traffic is established, well-known ports, seems ok. But once webrtc is involved, I couldn't determine which ports should be forwarded through HAProxy. There should be a range of ports used by OME and it must be selecting the port (possibly random one) for webrtc session. I can set HAProxy to forward range of ports, and if possible, I should narrow it to prevent conflict with other services.

Could you please provide some information, what is that range, or which settings/parameter controls that (if any), or any other means that I could set that range? Thank you.

**Comment by getroot** [comment]
Is this the information you are looking for?
https://airensoft.gitbook.io/ovenmediaengine/streaming/webrtc-publishing#configuration
